src/domains/dashboard/sagas.ts

```diff
@@ -32,16 +36,26 @@ function resetGlobalPanAndZoomSaga() {
 }
 
 function setGlobalChartUnderlaySaga({ payload }: Action<SetGlobalChartUnderlayAction>) {
+  const { after, before } = payload
   if (window.urlOptions) {
-    const { after, before } = payload
     // additional check to prevent loop, after setting initial state from url
     if (window.urlOptions.after !== after || window.urlOptions.before !== before) {
       window.urlOptions.netdataHighlightCallback(true, after, before)
     }
+  } else {
+    // TODO: Consider a setting to control wether the component sets these hash params
+    const hashParams = getHashParams()
+    const highlightAfter = new Date(Math.round(after)).toJSON()
+    const highlightBefore = new Date(Math.round(before)).toJSON()
```

Comment:
nice idea, this format will be much friendlier than timestamps. Perhaps we should also remove microseconds from the hash string?